numpy/lib/function_base.py

```diff
@@ -4326,6 +4327,69 @@ def meshgrid(*xi, **kwargs):
             return np.broadcast_arrays(*output)
 
 
+def meshgridify(*xi, **kwargs):
+    """
+    Converts tabular data to meshgrid format.
+
+    Returns array of function values `f` fitting to meshgrid arrays created
+    from `xi`. If no `f` is given, only a meshgrid from `xi` is returned.
+
+    Parameters
+    ----------
+    x1, x2,..., xn : array_like
```

Comment:
Since meshgridify calls the meshgrid function with **kwargs when no `f` is given it coded it like that.
But of course an api with only (*xi) and f being maybe xi[0] would also be an option.